treeherder/webapp/api/push.py

```diff
@@ -213,20 +203,26 @@ def health_summary(self, request, project):
         try:
             push = Push.objects.get(revision=revision, repository__name=project)
         except Push.DoesNotExist:
-            return Response("No push with revision: {0}".format(revision),
-                            status=HTTP_404_NOT_FOUND)
+            return Response(
+                "No push with revision: {0}".format(revision), status=HTTP_404_NOT_FOUND
+            )
 
         push_health_test_failures = get_test_failures(push)
         push_health_lint_failures = get_lint_failures(push)
         push_health_build_failures = get_build_failures(push)
-
-        return Response({
-            'needInvestigation':
-                len(push_health_test_failures['needInvestigation']) +
-                len(push_health_build_failures) +
-                len(push_health_lint_failures),
-            'unsupported': len(push_health_test_failures['unsupported']),
-        })
+        test_failure_count = len(push_health_test_failures['needInvestigation'])
```

Comment:
This chunk is the only actual code-change.  I added a bit more detail to this endpoint so that the new `PushHealthSummary` component can give a bit more detail.